jarviscli/CmdInterpreter.py

```diff
@@ -83,6 +84,7 @@ def __init__(self, first_reaction_text, prompt, first_reaction=True, enable_voic
                         "umbrella",
                         {"update": ("location", "system")},
                         "weather",
+                        "fb_login"
```

Comment:
Ok I will correct it and will keep that in mind in future. And underscore is not the problem I think because there are functions similar to it (having underscore).